Formula/gwyddion.rb

```diff
@@ -21,6 +22,10 @@ class Gwyddion < Formula
   depends_on "pygtk"
   depends_on "python@2"
 
+  # Fixes problem with finding resource files in version 2.53.
+  # <https://sourceforge.net/p/gwyddion/mailman/message/36604431/>
+  patch :DATA
```

Comment:
Please use upstream patch URL (http://gwyddion.net/download/2.53/gwyddion-2.53-ensure-osx-basedir.patch) rather than `:DATA`.

Reply:
Just updated the PR with the upstream patch.